packages/hoppscotch-common/src/components/collections/ImportExport.vue

```diff
@@ -355,7 +406,9 @@ const HoppGistImporter: ImporterOrExporter = {
   component: GistSource({
     caption: "import.from_url",
     onImportFromGist: async (content) => {
-      if (E.isLeft(content)) {
+      isGistImporterInProgress.value = true
```

Comment:
Updates to `HoppGistImporter` (toggling state variable and `try/finally` block) can be reverted since the loading state is handled at the component source.